From 0ccdb96b26903f46d0eb5d007986fe22f54b1819 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 06 Sep 2018 09:26:06 +0300
Subject: [PATCH] board: habana: scp preserve the modification time

While copying the scripts, binaries and other stuff,
preserve its modification date and time.
This is helpful to understand when the file has changed/built.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_run b/board/habanalabs/common/remote_run
index bcd73ef..9cd5dab 100755
--- a/board/habanalabs/common/remote_run
+++ b/board/habanalabs/common/remote_run
@@ -38,9 +38,9 @@
 	local scripts=(${scr_name[*]/local_/${local_scr_dir}/local_})
 	# sync the latest built binaries
 	for ((j=0; j<3; j++)); do
-		sshpass -p123456 scp ${fw_dir}/* root@${host}:${fw_dir}/	|| break;
-		sshpass -p123456 scp ${local_scr_dir}/${fw_ldr} ${host}:${remote_scr_dir}/	|| break;
-		sshpass -p123456 scp ${scripts[*]} ${host}:${remote_scr_dir}/	|| break;
+		sshpass -p123456 scp -p ${fw_dir}/* root@${host}:${fw_dir}/			|| break;
+		sshpass -p123456 scp -p ${local_scr_dir}/${fw_ldr} ${host}:${remote_scr_dir}/	|| break;
+		sshpass -p123456 scp -p ${scripts[*]} ${host}:${remote_scr_dir}/		|| break;
 		print_msg "Finished copying the firmware and scripts" | tee -a $result
 		return 0;
 	done
